### PR TITLE
Issue 966 new scipy

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

**Comment by doutriaux1** [comment]
@remram44 please test and let us know if that fixes your issue

**Comment by remram44** [comment]
SciPy built alright, I'm doing the full build and will run ctest.

**Comment by remram44** [comment]
[`100% tests passed, 0 tests failed out of 330`](https://open.cdash.org/buildSummary.php?buildid=3744080)

File: CMake/cdat_modules/scipy_pkg.cmake [hunk 1]
```diff
@@ -1,16 +1,16 @@
 set(SCIPY_MAJOR 0)
 set(SCIPY_MINOR 10)
 set(SCIPY_PATCH 0)
-set(SCIPY_MAJOR_SRC 0)
-set(SCIPY_MINOR_SRC 15)
-set(SCIPY_PATCH_SRC 1)
+set(SCIPY_MAJOR_SRC dde07a64407ffaa9442b3d8298c6c26ff91fb384)
```
Comment (aashish24) [inline CMake/cdat_modules/scipy_pkg.cmake]: Is this SCIPY available on LLNL sever? what does this number represents?

Comment (remram44) [inline CMake/cdat_modules/scipy_pkg.cmake]: I think a specific version is obtained from SciPy's Github, not just the master at the time of build. See [scipy/scipy:dde07a6](https://github.com/scipy/scipy/tree/dde07a64407ffaa9442b3d8298c6c26ff91fb384)

**Pull request merged**